tests/python/unittest/test_sparse_ndarray.py

```diff
@@ -962,6 +962,11 @@ def test_sparse_nd_check_format():
     indptr_list = [0, -2, 2, 3]
     a = mx.nd.sparse.csr_matrix((data_list, indices_list, indptr_list), shape=shape)
     assertRaises(mx.base.MXNetError, a.check_format)
+    # CSR format should be 2 Dimensional.
+    a = mx.nd.array([1, 2, 3]).tostype('csr')
+    assertRaises(mx.base.MXNetError, a.check_format)
```

Comment:
I think you want to do `a = mx.nd.array([1, 2, 3]); assertRaises(ValueError, a.tostype, 'csr') `instead?

Reply:
Will make the change.